src/language/templates/pybind/pyast.cpp

```diff
@@ -9,10 +9,11 @@
 /// THIS FILE IS GENERATED AT BUILD TIME AND SHALL NOT BE EDITED.
 ///
 
+#include "pybind/pyast.hpp"
```

Comment:
Don't we put project-internal includes _after_ external includes?

Reply:
TLDR: it has changed last week :)
We should now use [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) conventions about header inclusion.

This is something @pramodk and I discussed [here](https://github.com/BlueBrain/nmodl/pull/310#discussion_r417095638)

* there will be a PR in nmodl shortly to update all compilation units accordingly
* we should communicate about in BBP, at least within HPC